spec/integration/machine_image_spec.rb

```diff
@@ -25,6 +25,42 @@
           name: 'test_machine_image'
         ).and be_idempotent
       end
+
+      describe 'action :destroy' do
```

Comment:
This should be `describe 'action :destroy', :super_slow do` to mark this test as super slow